tests/unit/single_table/test_base.py

```diff
@@ -264,7 +264,7 @@ def test_get_metadata(self):
         result = instance.get_metadata()
 
         # Assert
-        assert result == metadata
+        assert result.to_dict() == metadata.to_dict()
```

Comment:
Metadata is now a deepcopy of the given object, so the ids are not the same anymore.